src/CustomComposerHelper.php

```diff
@@ -0,0 +1,124 @@
+<?php
```

Comment:
helpers should be grouped in folder "helpers"

Class name doesn't explain what it is for. Actually these should be 2 different helpers:
1) ScriptHelper with parse_arguments() method. (You can also add code, which is related to general processing of script, like some getters from composer variables, etc)
2) FileSystemHelper